Quantum-Control-Applications-QUAM/superconducting/calibrations/README.md

```diff
@@ -0,0 +1,79 @@
+# Calibration Graph Overview
+
+This document provides an overview of the calibration nodes and the concept of calibration graphs within the Qualibrate framework used in this project.
+
+## Collection of Calibration Nodes
+
+The `calibrations` directory contains a collection of standardized Python scripts, each representing a **Calibration Node**. Think of each node as a single, well-defined step in the overall process of calibrating and characterizing qubits or other quantum elements.
+
+A typical calibration node performs the following workflow:
+
+1.  **Loads QUAM State:** Reads the current state of the quantum machine configuration (QUAM).
+2.  **Executes QUA Program:** Runs a specific QUA sequence tailored to the calibration task (e.g., spectroscopy, Rabi, T1).
+3.  **Performs Analysis:** Processes the acquired data and performs fitting or other analysis to extract relevant parameters.
+4.  **Updates QUAM State:** Based on the analysis, it proposes changes to the relevant QUAM parameters (e.g., updating a frequency, amplitude, or duration).
+5.  **Saves Results:** Persists the input parameters, raw/processed data, analysis results, generated plots, and the proposed QUAM state updates for review and tracking.
+
+## Node Anatomy Explained
```

Comment:
Maybe we should explain here the attributes of the node object that the user can use, such as node.machine, node.results, node.namespace..., what do you thing?

Reply:
Done!